integrations/terraform/testlib/fixtures/bot_v2_1_update.tf

```diff
@@ -0,0 +1,34 @@
+locals {
+  bot_name = "test"
+}
+
+resource "teleport_provision_token" "bot_test" {
+  version = "v2"
+  metadata = {
+    expires = "2038-01-01T00:00:00Z"
+    name    = "bot-test"
+  }
+
+  spec = {
+    roles       = ["Bot"]
+    bot_name    = local.bot_name
+    join_method = "token"
+  }
+}
+
+resource "teleport_bot_v2" "test" {
```

Comment:
Could we extend the test/example to cover the TTL field as well? I know in the past Durations have been problematic with Terraform.